content/topics/javascript-specific/js-naming-convention/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+content_type: topic
+prerequisites:
+  hard:
+    - topics/file-and-directory-naming/general # This is currently missing. It is still here: Umuzi-org/ACN-syllabus#314
+
+ready: true
+tags:
+  - File and directory naming
+title: File and directory naming in JavaScript
+---
+
+## How should I name my JavaScript files and directories?
+
+When thinking about how to name files in JavaScript, think of the file as an entity within the code. This means, if you have a file with a class called `MyClass`, then it also makes sense to name the file `MyClass.js` ; notice that the naming convention utilizes _PascalCase_. The same thing for a file containing a function. `myFunction` should live inside a _camelCased_ file called `myFunction.js` . The same logic should apply for naming directories that live inside your **src** directory. Remember to be always consistent.
```

Comment:
This paragraph is a mouth full, perhaps have two paragraphs. Also recruits fall into the trap of naming stuff mySomething especially in early stages? I can we modify this content such that humans don't think its fine to name stuff mySomething.